plasmapy/classes/tests/test_Plasma.py

```diff
@@ -111,10 +121,23 @@ def test_intermediate_and_both_energies(self):
         regimes = ['Intermediate coupling regime: Gamma = 10.585076050938532.',
                    'Both Fermi and thermal energy important: Theta = 0.04113394941084522']
 
-        assert blob.regimes() == regimes
+        testTrue = blob.regimes() == regimes
 
+        errStr = (f"Regimes \n{regimes}\n should be equal to that of class\n"
+                  f"{blob.regimes()}.")
+        assert testTrue, errStr
 
     def test_strong_and_fermi(self):
+        r"""
+        Method to test Gamma and Theta values on PlasmaBlob.regimes()
+
+        Tests that input values return a strongly coupled regime
```

Comment:
Same problem here. This should be something like "Tests against expected parameters for a plasma which is simultaneously in the strongly coupled and (Fermi) degenerate regimes."